themes/unl/templates/RecordDriver/SolrDefault/core.phtml

```diff
@@ -94,27 +105,13 @@
         </td>
       </tr>
       <? endif; ?>
-	  
```

Comment:
Never mind - this come back in later commits so I'm not worrying about it here...